xml/System.Threading/ThreadPool.xml

```diff
@@ -1388,7 +1388,7 @@ If a thread pool implementation may have different types of threads, the count i
         <typeparam name="TState">The type of elements of <paramref name="state" />.</typeparam>
         <param name="callBack">A delegate representing the method to execute.</param>
         <param name="state">An object containing data to be used by the method.</param>
-        <param name="preferLocal">`true` to execute in the local threadpool; `false` to execute in the global thread pool.</param>
+        <param name="preferLocal"><see langword="true" /> to prefer queueing the work item in a queue close to the current thread; <see langword="false" /> to prefer queueing the work item to the thread pool's shared queue.</param>
         <summary>Queues a method specified by an <see cref="T:System.Action`1" /> delegate for execution, and specifies an object containing data to be used by the method. The method executes when a thread pool thread becomes available.</summary>
         <returns>
           <see langword="true" /> if the method is successfully queued; <see cref="T:System.NotSupportedException" /> is thrown if the work item could not be queued.</returns>
```

Comment:
An exception could be listed after the \<remarks> element:
- <exception cref="T:System.ArgumentNullException"><paramref name="callback" /> is <see langword="null" />.</exception>

Reply:
We cannot add a suggestion for line 1406 (too far from the closest change). This is what @rpetrusha suggested, plus the exception mentioned in the `<returns>`:

```xml
        <remarks>To be added.</remarks>
        <exception cref="T:System.ArgumentNullException"><paramref name="callback" /> is <see langword="null" />.</exception>
        <exception cref="T:System.NotSupportedException">The work item could not be queued.</exception>
```